examples/BullhornExampleDefaultSchema.html

```diff
@@ -0,0 +1,94 @@
+<!--
+Simple sample page to demonstrate the use of topic-based messaging with bullhorn.
+It retrieves three instances of a channel defined without setting a schema (a default one will be used), publishes on one and
+subscribes on the other two.
+It is intended to be loaded directly from the file system in this directory.
+Most of the code is setting up AMD (which becomes negligible in larger projects).
+-->
+<!DOCTYPE html>
+<html>
+<head>
+    <!--
+    Include only an amd loader.
+    -->
+    <script src="require.js"></script>
+    <script>
+        // Configure AMD to find things.
+        var path = location.pathname.replace(/\/[^/]+$/, '');
+        require.config({
+            baseUrl: "./",
+            paths: {
+                bullhorn: "../js",
+                schema: "schema", // the example schema
+                example: "."
+            },
+            // remap to use a third party validator.
+            map: {
+                "*": {
+                    "bullhorn/Validator": "example/Validator"
+                }
+            },
+            config: {
+                "bullhorn/log": {
+                    logLevel: "debug"
+                }
+            }
+        });
+    </script>
+    <script>
+        // Use AMD to define and load the resources we depend on.
+        require([
+            "bullhorn/ChannelFactory",
+            "schema/ExampleChannel"
+        ], function (
+                ChannelFactory,
+                ExampleChannel
+        ) {
+            // Create a channel factory.
+            var cfact = new ChannelFactory({
+                    // "resolver" is used by the factory to find schemas based on names.
+                    // resolve schema by mapping it to the dependency where it was already loaded.
+                    resolver: function (name) {
```

Comment:
With the new default schema-less operation, can we omit this resolver function entirely?